buildkite/bazelci.py

```diff
@@ -1351,6 +1351,11 @@ def PrepareRepoInCwd(print_cmd_groups, initial_setup=False):
 
         execute_bazel_run(bazel_binary, platform, task_config.get("run_targets", None))
 
+        if platform == "windows":
```

Comment:
I think this should be moved to line 1520/1525, basically after the `index_targets`

Reply:
lol makes total sense. Moved!